openid-4-verifiable-presentations-1_0.md

```diff
@@ -432,54 +432,32 @@ Note: "https://self-issued.me/v2" is a symbolic string and can be used as an `au
 
 ## Verifier Metadata Management {#client_metadata_management}
 
-The `client_id_scheme` enables deployments of this specification to use different mechanisms to obtain and validate metadata of the Verifier beyond the scope of [@!RFC6749]. The term `client_id_scheme` is used since the Verifier is acting as an OAuth 2.0 Client.
+If the Client is not pre-registered with the Wallet, the `client_id` MUST be a URI.
 
-This specification defines the following values for the `client_id_scheme` parameter, followed by the examples where applicable: 
+The Wallet SHOULD verify the provided `redirect_uri` or `response_uri` belongs to the Client.
```

Comment:
@selfissued: it is a SHOULD because for DIDs there is no obvious way how to do that, even with the existing OID4VP text.

Reply:
I'd make it a MUST.  And then maybe add this comment:
(Note that it is currently unspecified how to achieve this for DIDs.)